lib/config.js

```diff
@@ -39,8 +39,8 @@ const conf = convict({
     arg: 'node-env',
     doc: 'The current node.js environment',
     env: 'NODE_ENV',
-    format: ['dev', 'test', 'stage', 'prod'],
-    default: 'dev'
+    format: ['development', 'test', 'stage', 'production'],
+    default: 'development'
```

Comment:
Looks like auth-server and content-server default to `production`. Should this default to it as well?